File: lat2/python/main.py
from timsepakbola import DaftarTimSepakbola

# menggunakan setter
sepakbola1 = DaftarTimSepakbola()
sepakbola1.setNamaTim("Persebaya");
sepakbola1.setAsalNegara("Indonesia");
sepakbola1.setTahunTim(1927);
sepakbola1.setNamaPemain("Ernando Ari");
sepakbola1.setNomorPemain(1);
sepakbola1.setPosisi("Goal Keeper");

sepakbola2 = DaftarTimSepakbola()
sepakbola2.setNamaTim("Bali United");
sepakbola2.setAsalNegara("Indonesia");
sepakbola2.setTahunTim(1989);
sepakbola2.setNamaPemain("Nadeo Argawinata");
sepakbola2.setNomorPemain(23);
sepakbola2.setPosisi("Goal Keeper");

# menggunakan constructor
sepakbola3 = DaftarTimSepakbola("Borneo FC", "Indonesia", 2014, "Marckho Sandy", 2, "Defender")

# tampilkan daftar tim sepakbola ke layar
# tim sepakbola 1
print("Daftar Tim sepakbola")
print(" ")
print("1")
print("Nama Tim : " + str(sepakbola1.getNamaTim()))
print("Asal Negara : " + str(sepakbola1.getAsalNegara()))
print("Tahun Berdiri Tim : " + str(sepakbola1.getTahunTim()))
print("Nama Pemain : " + str(sepakbola1.getNamaPemain()))
print("Nomor Pemain : " + str(sepakbola1.getNomorPemain()))
print("Posisi : " + str(sepakbola1.getPosisi()))

# tim sepakbola 2
print(" ")
print("2")
print("Nama Tim : " + str(sepakbola2.getNamaTim()))
print("Asal Negara : " + str(sepakbola2.getAsalNegara()))
print("Tahun Berdiri Tim : " + str(sepakbola2.getTahunTim()))
print("Nama Pemain : " + str(sepakbola2.getNamaPemain()))
print("Nomor Pemain : " + str(sepakbola2.getNomorPemain()))
print("Posisi : " + str(sepakbola2.getPosisi()))

# tim sepakbola 3
print(" ")
print("3")
print("Nama Tim : " + str(sepakbola3.getNamaTim()))
print("Asal Negara : " + str(sepakbola3.getAsalNegara()))
print("Tahun Berdiri Tim : " + str(sepakbola3.getTahunTim()))
print("Nama Pemain : " + str(sepakbola3.getNamaPemain()))
print("Nomor Pemain : " + str(sepakbola3.getNomorPemain()))
print("Posisi : " + str(sepakbola3.getPosisi()))

File: lat2/cpp/main.cpp
#include <iostream>
#include <string>
using namespace std;
#include "timsepakbola.cpp"

int main(){

	DaftarTimSepakbola sepakbola1;
	DaftarTimSepakbola sepakbola2;
	DaftarTimSepakbola sepakbola3;

	//set dan get

	sepakbola1.setNamaTim("Persebaya");
	sepakbola1.setAsalNegara("Indonesia");
	sepakbola1.setTahunTim(1927);
	sepakbola1.setNamaPemain("Ernando Ari");
	sepakbola1.setNomorPemain(1);
	sepakbola1.setPosisi("Goal Keeper");

	cout << "Daftar Tim Sepakbola" << endl;
	cout << "1" << endl;
	cout << sepakbola1.getNamaTim() << endl;
	cout << sepakbola1.getAsalNegara() << endl;
	cout << sepakbola1.getTahunTim() << endl;
	cout << sepakbola1.getNamaPemain() << endl;
	cout << sepakbola1.getNomorPemain() << endl;
	cout << sepakbola1.getPosisi() << endl;

	sepakbola2.setNamaTim("Bali United");
	sepakbola2.setAsalNegara("Indonesia");
	sepakbola2.setTahunTim(1989);
	sepakbola2.setNamaPemain("Nadeo Argawinata");
	sepakbola2.setNomorPemain(23);
	sepakbola2.setPosisi("Goal Keeper");

	cout << "\n" << endl;
	cout << "2" << endl;
	cout << sepakbola2.getNamaTim() << endl;
	cout << sepakbola2.getAsalNegara() << endl;
	cout << sepakbola2.getTahunTim() << endl;
	cout << sepakbola2.getNamaPemain() << endl;
	cout << sepakbola2.getNomorPemain() << endl;
	cout << sepakbola2.getPosisi() << endl;


	cout << "\n" << endl;
	cout << "3" << endl;
	sepakbola3.setNamaTim("Borneo FC");
	sepakbola3.setAsalNegara("Indonesia");
	sepakbola3.setTahunTim(2014);
	sepakbola3.setNamaPemain("Marckho Sandy");
	sepakbola3.setNomorPemain(2);
	sepakbola3.setPosisi("Defender");

	cout << sepakbola3.getNamaTim() << endl;
	cout << sepakbola3.getAsalNegara() << endl;
	cout << sepakbola3.getTahunTim() << endl;
	cout << sepakbola3.getNamaPemain() << endl;
	cout << sepakbola3.getNomorPemain() << endl;
	cout << sepakbola3.getPosisi() << endl;

	return 0;
}